.github/workflows/build-test.yml

```diff
@@ -0,0 +1,176 @@
+name: Build Test
+
+on:
+  pull_request:
+    paths-ignore:
+    - README.md
+    - VERSION_INFO
+    - .ci/*
+    - .readthedocs.yml
+    - docs-src/*
+    - docs-img/*
+    - docs-jupyter/*
+    - docs-doxygen/*
+    - docs-sphinx/*
+    - docs/*
+    - studies/*
+    - src/cuda-kernels
+    - cuda-kernels
+  # Don't run for any PR
+  schedule:
+  - cron: '0 12 1 * *'
+
+  workflow_dispatch:
+
+concurrency:
+  group: build-test-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  Windows:
+    strategy:
+      matrix:
+        python-version:
+        - '3.9'
+        - '3.10'
+        - '3.9'

```

Comment:
`3.9` dup?

Reply:
Ah - missed the arch :warning: 

Do you know why we had an x86 test? @jpivarski?